standard/documentation-comments.md

```diff
@@ -61,6 +61,16 @@ Although developers are free to create their own set of tags, a recommended set
 
 Note carefully that the documentation file does not provide full information about the type and members (for example, it does not contain any type information). To get such information about a type or member, the documentation file must be used in conjunction with reflection on the type or member.
 
+A partial type or a partial non-type member (such as a method) can be declared in multiple parts, each of which can be in one or more compilation units, and each of which can have one or more documentation comments. A partial non-type member typically has a “defining partial declaration” and an “implementing partial declaration.”
```

Comment:
Do we have any partial non-type members other than methods at the moment? If not, let's just use "or a partial method" for now.

Reply:
Other partial members will become important for our future selves, when we get to C# 13.